src/test/obj_check/obj_check.c

```diff
@@ -48,8 +48,31 @@ main(int argc, char *argv[])
 	}
 
 	const char *path = argv[1];
+	const char *layout = NULL;
+	PMEMobjpool *pop;
+	int open = 0;
+	int close = 0;
 
-	int ret = pmemobj_check(path, NULL);
+	for (int i = 0; i < argc; ++i) {
```

Comment:
IMHO, You could start iterate here from i=2.
If You do this, You could also add else condition, below to detect unrecognized parameter ( different than **-o** and **-l**)